extensions/amp-live-list/0.1/poller.js

```diff
@@ -107,14 +109,16 @@ export class Poller {
   /**
    * Queues a timeout that executes the work and recursively calls
    * itself on success.
+   * @param {boolean=} opt_isLeading execute current work instead of queueing
+   *     it in a timeout.
    * @private
    */
-  poll_() {
+  poll_(opt_isLeading) {
```

Comment:
took the nomenclature from http://underscorejs.org/#throttle (where it has a "leading" flag), i could change it to "immediate" which is the flag they use for debounce as that might seem clearer